pkg/helm/build.go

```diff
@@ -50,6 +46,7 @@ type BuildInput struct {
 //		  username: "username"
 //		  password: "password"
 type MixinConfig struct {
+	Version      string `yaml:"version,omitempty"`
```

Comment:
I just had a thought.  In the future, I believe we'll want to enable the user specifying the actual mixin version, which would probably be called `Version`.  Perhaps we change this (and the corresponding yaml field) to `HelmClientVersion` (`helmClientVersion`) or something other than `Version`.  WDYT?

Reply:
How about clientVersion?